coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/TestUtils.java

```diff
@@ -357,6 +357,10 @@ public static RelNode convertView(String db, String view) {
     return new HiveToRelConverter(hiveMetastoreClient).convertView(db, view);
   }
 
+  public static RelNode convertView(String sql) {
+    return new HiveToRelConverter(hiveMetastoreClient).convertSql(sql);
+  }
+
```

Comment:
Why do we need this method?

Reply:
I would prefer to remove the method if it's not used, and we can add if we need it in the future.

Reply:
ack. will address.